DynamicSmoothedGUI.py

```diff
@@ -169,6 +171,56 @@ def _color_normally(r, angle_rad):
                    _color_normally(r, angle_rad)
                lidar_data_copy.pop(0)
 
+    def drawC1C0(self):
+        """Draws C1C0's current location on the simulation"""
+
+        def _scale_coords(coords):
+            """scales coords (a tuple (x, y)) from real life cm to pixels"""
+            scaled_x = coords[0] / tile_scale_fac
+            scaled_y = coords[1] / tile_scale_fac
+            scaled_coords = (scaled_x, scaled_y)
```

Comment:
I would just return (scale_x, scale_y) extra variable adds LOC and does not make it more readable imo